app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -246,7 +244,7 @@ private void setUpMenu() {
         captureButton = menu.findViewById(R.id.capture_page);
         refreshIcon = menu.findViewById(R.id.action_refresh);
         stopIcon = menu.findViewById(R.id.action_stop);
-        menu.findViewById(R.id.menu_turbomode).setSelected(mTurboModePref);
+        menu.findViewById(R.id.menu_turbomode).setSelected(isTurboEnabled());
```

Comment:
Suggest to do the same (remove the caching) for blockimage too, just to keep just one logic for similar things which hopefully makes the life of future developers easier...